master/internal/user/postgres_users.go

```diff
@@ -117,6 +117,23 @@ func Update(
 	})
 }
 
+// SetActive changes multiple users' activation status.
+func SetActive(
+	ctx context.Context,
+	updateIDs []model.UserID,
+	activate bool,
+) error {
+	if len(updateIDs) > 0 {
+		if _, err := db.Bun().NewUpdate().
+			Table("users").
+			Set("active = ?", activate).
+			Where("id IN (?)", bun.In(updateIDs)).Exec(ctx); err != nil {
+			return fmt.Errorf("error updating %q: %s", updateIDs, err)
```

Comment:
this is a super nit but you include setting active in the error message?

Reply:
makes sense